package.json

```diff
@@ -8,9 +8,12 @@
   "repository": "https://github.com/thysultan/stylis.js",
   "bugs": "https://github.com/thysultan/stylis.js/issues",
   "type": "module",
-  "main": "dist/stylis.umd.js",
+  "main": "dist/umd/stylis.umd.js",
   "module": "dist/stylis.esm.js",
-  "exports": "./index.js",
+  "exports": {
+    "import": "./index.js",
```

Comment:
I personally don't quite like that a different file is being given to node (pkg.exports.import) and bundlers (pkg.module) - if the code is not specific to its environment.

Could we just point to the same file from both? I'm still not a fan of giving bundlers pre-minified code - had to debug few 3rd part dependencies in the past weeks and this would be a total pain if they would be minified (luckily, they weren't).

Reply:
We still want to point main to the umd file.

Reply:
Just because conceptually I see `pkg.main` as `pkg.exports.require`, so I've reused the same file - it also avoids UMD wrapper for node@<13 consumers, not a big deal - can change it back if you want.